doc/src/installation.md

```diff
@@ -122,6 +122,77 @@ Before the first build you have to set up the `ZLib`-pathes: Go to the CMake-Set
 
 Then you can generate the CMake-Cache and build the project.
 
+## Build from Git on Windows with WSL
+* In windows search "turn windows features on or off".
+* Tick Windows subsystem for Linux.
+* Press OK and restart computer.
+* Go to Microsoft store and download Ubuntu 22.04, and launch it.
+* Run the following commands.
+
+```bash
+wget  https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O miniconda.sh
+bash miniconda.sh -b -p $HOME/conda_root
+export PATH="$HOME/conda_root/bin:$PATH"
+```
+* Now do the following to configure the path
+```bash
+sudo nano .bashrc
+```
+* Then go to the bottom of the file and paste the following
+```bash
+export PATH="$HOME/conda_root/bin:$PATH"
+```
+* Then press ctrl + O (save), Enter (confirm), ctrl + X (exit)
+* After that restart Ubuntu
+* Run the following
+```bash
+conda create -n lp -c conda-forge llvmdev=11.0.1 bison=3.4 re2c python cmake make toml
+conda init bash
+```
+* Note that Conda is slow for installing packages with specified versions.
+* Instead use Mamba for faster installation, with the following commands
+```bash
+conda create -n lp -c conda-forge re2c python cmake make toml numpy
+conda install mamba -c conda-forge
+mamba install llvmdev=11.0.1 bison=3.4
+conda init bash
+```
+* Restart Ubuntu again
+```bash
+conda activate lp
+sudo apt update
+sudo apt-get install build-essential
+sudo apt-get install zlib1g-dev
+sudo apt install clang
```

Comment:
Use conda for installing `clang`?

Reply:
To be honest, I don't know which command is better. My own journey was experimental.

Reply:
> Use conda for installing `clang`?

Do you mean this - `conda install -c conda-forge clangdev`  ?

Reply:
Yes. `sudo apt` installs things system wide which is not recommended. Once you have a conda environment then everything should be kept localised to that environment with the help of `conda install`.